pylint/utils.py

```diff
@@ -810,7 +815,9 @@ def expand_modules(files_or_modules, black_list, black_list_re):
     result = []
     errors = []
     for something in files_or_modules:
-        if exists(something):
+        if _path_delimiter() in something:
```

Comment:
Please write a comment here describing why exactly is this needed. Also, move the condition after exists(something) and before the bare else clause.